src/lib/defaults.js

```diff
@@ -36,33 +36,39 @@ module.exports = {
   INCLUDE_DIRECTIVE: '$include',
   APPLICATION_CONFIG_KEY: 'application',
   EXTENSIONS_CONFIG_KEY: 'extensions',
-  implPromptChoices: [
-    // we abuse the extension command to also let users add a standalone app
+  EXTENSION_POINT_LIST: [ // Extension point list for easy access to all supported extension points
+    'dx/excshell/1',
+    'dx/asset-compute/worker/1'
+  ],
+  // we still need some details for extensions, we add them to the promt choices later
+  extensionDefaults: {
+    'dx/excshell/1':
     {
-      name: 'Standalone Application',
-      value: {
-        name: 'application',
-        generator: generators.application,
-        requiredServices: [] // TODO required services should be filled based on selected actions
-      }
-    },
-    // extensions
-    // TODO this list should not be hardcoded but fetched from xt reg
-    {
-      name: 'DX Experience Cloud SPA v1',
+      name: 'Firefly Experience Cloud Shell',
       value: {
         name: 'dx/excshell/1',
         generator: generators.extensions['dx/excshell/1'],
         requiredServices: []
       }
     },
-    {
+    'dx/asset-compute/worker/1': {
       name: 'DX Asset Compute Worker v1',
       value: {
         name: 'dx/asset-compute/worker/1',
         generator: generators.extensions['dx/asset-compute/worker/1'],
         requiredServices: ['AssetComputeSDK']
       }
     }
+  },
+  implPromptChoices: [
```

Comment:
This is not very clean, implPromptChoices should include both extensions and standalone app, otherwise please rename that default